lib/internal/http2/core.js

```diff
@@ -1506,6 +1506,13 @@ class Http2Stream extends Duplex {
       this.once('ready', this._destroy.bind(this, err, callback));
       return;
     }
+
+    const server = session[kServer];
+
+    if (err && server) {
+      server.emit('clientError', err, this);
```

Comment:
I think so - or at least I'd expect it to. Do we have a guarantee that it will always be deferred by the time _destroy is called?